diff --git a/src/dragonfly/Makefile b/src/dragonfly/Makefile
index db43493e3..bdcd2b8ee 100644
--- a/src/dragonfly/Makefile
+++ b/src/dragonfly/Makefile
@@ -8,7 +8,7 @@ EXTRA_LDLIBS = $(subst libfst,libfstscript,$(OPENFSTLIBS))
 
 TESTFILES =
 
-OBJFILES = base-nnet3.o agf-sub-nnet3.o plain-sub-nnet3.o laf-sub-nnet3.o fst-export.o md5.o
+OBJFILES = base-nnet3.o active-base-nnet3.o agf-sub-nnet3.o plain-sub-nnet3.o laf-sub-nnet3.o fst-export.o md5.o
 
 LIBNAME = kaldi-dragonfly
 DynamicLibrary = kaldi-dragonfly
